[2022-06-06 09:57:04,338] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T00:25:00+00:00 [queued]>
[2022-06-06 09:57:04,343] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T00:25:00+00:00 [queued]>
[2022-06-06 09:57:04,343] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 09:57:04,343] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-06 09:57:04,343] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 09:57:04,354] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): tweets_to_es> on 2022-06-06 00:25:00+00:00
[2022-06-06 09:57:04,357] {standard_task_runner.py:52} INFO - Started process 6408 to run task
[2022-06-06 09:57:04,359] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'tweets_to_es', 'scheduled__2022-06-06T00:25:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/airfow_es.py', '--cfg-path', '/tmp/tmp79vpgjm6', '--error-file', '/tmp/tmp4hgb9pzc']
[2022-06-06 09:57:04,360] {standard_task_runner.py:80} INFO - Job 7: Subtask tweets_to_es
[2022-06-06 09:57:04,390] {task_command.py:370} INFO - Running <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T00:25:00+00:00 [running]> on host oury-Tp
[2022-06-06 09:57:04,438] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=data_pipeline
AIRFLOW_CTX_TASK_ID=tweets_to_es
AIRFLOW_CTX_EXECUTION_DATE=2022-06-06T00:25:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-06T00:25:00+00:00
[2022-06-06 09:57:04,439] {twitter.py:696} INFO - Retrieving scroll page None
[2022-06-06 09:57:04,439] {twitter.py:626} INFO - Retrieving guest token
[2022-06-06 09:57:04,440] {base.py:171} INFO - Retrieving https://twitter.com/search?f=live&lang=en&q=near%3ATunis+since%3A1654505524+until%3A1654505824&src=spelling_expansion_revert_click
[2022-06-06 09:57:04,937] {base.py:189} INFO - Retrieved https://twitter.com/search?f=live&lang=en&q=near%3ATunis+since%3A1654505524+until%3A1654505824&src=spelling_expansion_revert_click: 200
[2022-06-06 09:57:04,939] {base.py:171} INFO - Retrieving https://api.twitter.com/2/search/adaptive.json?include_profile_interstitial_type=1&include_blocking=1&include_blocked_by=1&include_followed_by=1&include_want_retweets=1&include_mute_edge=1&include_can_dm=1&include_can_media_tag=1&skip_status=1&cards_platform=Web-12&include_cards=1&include_ext_alt_text=true&include_quote_count=true&include_reply_count=1&tweet_mode=extended&include_entities=true&include_user_entities=true&include_ext_media_color=true&include_ext_media_availability=true&send_error_codes=true&simple_quoted_tweets=true&q=near%3ATunis+since%3A1654505524+until%3A1654505824&tweet_search_mode=live&count=100&query_source=spelling_expansion_revert_click&pc=1&spelling_corrections=1&ext=mediaStats%2ChighlightedLabel
[2022-06-06 09:57:05,632] {base.py:189} INFO - Retrieved https://api.twitter.com/2/search/adaptive.json?include_profile_interstitial_type=1&include_blocking=1&include_blocked_by=1&include_followed_by=1&include_want_retweets=1&include_mute_edge=1&include_can_dm=1&include_can_media_tag=1&skip_status=1&cards_platform=Web-12&include_cards=1&include_ext_alt_text=true&include_quote_count=true&include_reply_count=1&tweet_mode=extended&include_entities=true&include_user_entities=true&include_ext_media_color=true&include_ext_media_availability=true&send_error_codes=true&simple_quoted_tweets=true&q=near%3ATunis+since%3A1654505524+until%3A1654505824&tweet_search_mode=live&count=100&query_source=spelling_expansion_revert_click&pc=1&spelling_corrections=1&ext=mediaStats%2ChighlightedLabel: 200
[2022-06-06 09:57:06,072] {logging_mixin.py:115} INFO - {'id': 1533734448801497090, 'username': 'FLASHMAGtn', 'tweet_date': datetime.datetime(2022, 6, 6, 8, 56, 2, tzinfo=datetime.timezone.utc), 'tweet_content': 'Est-ce que l’eau abîmé les cheveux\xa0? https://t.co/UOIiyVA2qw', 'likes_count': 0, 'tweet_url': 'https://twitter.com/FLASHMAGtn/status/1533734448801497090', 'tweet_lang': 'fr'}
[2022-06-06 09:57:06,074] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/oury/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/oury/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/home/oury/airflow/dags/airfow_es.py", line 49, in tweet_to_es_func
    index_es_func(tweets_index, tweet)
TypeError: index_es_func() missing 1 required positional argument: 'tweet'
[2022-06-06 09:57:06,077] {taskinstance.py:1396} INFO - Marking task as FAILED. dag_id=data_pipeline, task_id=tweets_to_es, execution_date=20220606T002500, start_date=20220606T085704, end_date=20220606T085706
[2022-06-06 09:57:06,086] {standard_task_runner.py:92} ERROR - Failed to execute job 7 for task tweets_to_es (index_es_func() missing 1 required positional argument: 'tweet'; 6408)
[2022-06-06 09:57:06,097] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-06 09:57:06,112] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2022-06-06 10:37:42,604] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T00:25:00+00:00 [queued]>
[2022-06-06 10:37:42,615] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T00:25:00+00:00 [queued]>
[2022-06-06 10:37:42,616] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 10:37:42,616] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-06 10:37:42,616] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 10:37:42,643] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): tweets_to_es> on 2022-06-06 00:25:00+00:00
[2022-06-06 10:37:42,647] {standard_task_runner.py:52} INFO - Started process 14536 to run task
[2022-06-06 10:37:42,653] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline', 'tweets_to_es', 'scheduled__2022-06-06T00:25:00+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/airfow_es.py', '--cfg-path', '/tmp/tmpt28q7oip', '--error-file', '/tmp/tmp0sxbir47']
[2022-06-06 10:37:42,655] {standard_task_runner.py:80} INFO - Job 7: Subtask tweets_to_es
[2022-06-06 10:37:42,726] {task_command.py:370} INFO - Running <TaskInstance: data_pipeline.tweets_to_es scheduled__2022-06-06T00:25:00+00:00 [running]> on host oury-Tp
[2022-06-06 10:37:42,780] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=data_pipeline
AIRFLOW_CTX_TASK_ID=tweets_to_es
AIRFLOW_CTX_EXECUTION_DATE=2022-06-06T00:25:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-06T00:25:00+00:00
[2022-06-06 10:37:42,781] {taskinstance.py:1890} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/oury/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/home/oury/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
TypeError: tweet_to_es_func() missing 5 required positional arguments: 'es', 'start', 'end', 'location', and 'languages'
[2022-06-06 10:37:42,785] {taskinstance.py:1396} INFO - Marking task as FAILED. dag_id=data_pipeline, task_id=tweets_to_es, execution_date=20220606T002500, start_date=20220606T093742, end_date=20220606T093742
[2022-06-06 10:37:42,800] {standard_task_runner.py:92} ERROR - Failed to execute job 7 for task tweets_to_es (tweet_to_es_func() missing 5 required positional arguments: 'es', 'start', 'end', 'location', and 'languages'; 14536)
[2022-06-06 10:37:42,825] {local_task_job.py:156} INFO - Task exited with return code 1
[2022-06-06 10:37:42,843] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
